app/src/main/java/swati4star/createpdf/util/FileUtils.java

```diff
@@ -419,4 +430,15 @@ String getUniqueFileName(String fileName) {
         }
         return outputFileName;
     }
+
+    public void makeAndClearTemp() {
```

Comment:
I would suggest making this method static.

Reply:
done